build.gradle

```diff
@@ -389,8 +392,21 @@ subprojects {
     }
   }
 
+  diffCoverageReport {
+    afterEvaluate {
+      diffSource.file = createDiffFile()
+    }
+
+    // Report locates at <module_name>/build/reports/jacoco/diffCoverage/html/index.html
+    reports {
+      html = true
+    }
 
-  check.dependsOn(jacocoTestCoverageVerification)
+    violationRules {
+      minBranches = 0.8
```

Comment:
Does it mean 80% test coverage on the diff?

Reply:
It's up to how we define test coverage. For this plugin, we can have branch coverage, line coverage and instruction coverage. From the last PR ([#157](https://github.com/linkedin/venice/pull/157#discussion_r1065063197)), we think branch coverage is more reasonable compared to other criteria, so that's why I chose branch coverage here.

We can also add other requirements if we think branch coverage is not sufficient.

Reply:
What is the current branch coverage?
I am wondering whether this criteria will fail every new commit or not in case the current branch coverage in master is lower than 0.8.

Reply:
I see, thanks.